packages/patternfly-4/react-core/src/components/Alert/Alert.md

```diff
@@ -174,3 +174,133 @@ class DangerAlert extends React.Component {
   }
 }
 ```
+
+## Static Live Region Alert
+```js
+import * as React from 'react';
+import { Alert } from '@patternfly/react-core';
+
+class StaticLiveRegionAlert extends React.Component {
+  render() {
+    return (
+      <React.Fragment>
+        <Alert toast={true} variant="info" title="Toast Alert Title"></Alert>
+        <Alert
+          aria-live="assertive"
```

Comment:
These aria attributes are spread to the Alert container div, but also the inner div has similar aria attributes?

Reply:
So it's actually an either or kinda thing. I think we want to recommend users just use the `isToast` prop which sets CSS classes as well as aria attributes to what we recommend in general. In this case the aria attributes are planted on the innermost element that contains the notification text.

There may be cases where a user needs more fine grain control over this experience, so in that case a user may omit the `isToast` prop, and set those attributes and classes manually. In this scenario users would be setting those aria attributes directory on the Notification container, which also allows for exposing other bits of content like the icon and actions, should a user want to do that.

It wouldn't be recommended to set both `isToast` and `aria-*` attributes at the same time. Does it make sense to put into place code to prevent this? Seems like it's not worth the added complexity, but I'd like to hear what others think.

In any case I updated the examples to make this more clear as I see how it could be confusing before. Let me know if you think this is enough for now, considering the full docs are still in the works.